config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -244,7 +244,7 @@ endif()
 # Define 'factory_data' target for generating a factory data partition
 # ==============================================================================
 
-if (CONFIG_CHIP_FACTORY_DATA_BUILD)
+if(CONFIG_CHIP_FACTORY_DATA_BUILD AND NOT SYSBUILD)
```

Comment:
Why skipping factory data when using sysbuild? Is it because of WIP?

Reply:
sysbuild code is in the other file

Reply:
Ok, I see the new file is used in sysbuild CMake.